_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,296 @@
+---
+title: 'Bitcoin Optech Newsletter #109'
+permalink: /en/newsletters/2020/08/05/
+name: 2020-08-05-newsletter
+slug: 2020-08-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains our regular sections with recently
+transcribed talks and conversations, releases and release candidates,
+and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **New spending policy language:** Nadav Ivgi [announced][ivgi minsc] a new
+  language he's developed named Minsc which makes it easier for
+  developers to specify a set of conditions that must be fulfilled in
+  order for a UTXO to be spent.  The new language is based on the
+  [miniscript][topic miniscript] policy language but adds the ability to
+  use variables and functions along with several other features.  Minsc
+  policies can be compiled to miniscript policies, which can be further transformed
```

Comment:
'further transformed' feels a bit vague here. My understanding is that policy is _compiled_ into miniscript, which can be converted into Bitcoin Script.

Reply:
This is correct. Perhaps use "which can be encoded to Bitcoin Script" instead of "converted into" https://bitcoin.stackexchange.com/questions/91565/what-does-bitcoin-policy-language-offer-the-developer-that-miniscript-doesnt-w

Reply:
Asking about this on [StackExchange](https://bitcoin.stackexchange.com/questions/98357/what-is-the-relationship-between-the-minsc-language-and-policy-language) and ##miniscript IRC channel to nail down the correct language.